CMakeLists.txt

```diff
@@ -339,8 +340,7 @@ IF (LIBSSH2_FOUND)
 	ADD_DEFINITIONS(-DGIT_SSH)
 	INCLUDE_DIRECTORIES(${LIBSSH2_INCLUDE_DIRS})
 	LINK_DIRECTORIES(${LIBSSH2_LIBRARY_DIRS})
-	LIST(APPEND LIBGIT2_PC_LIBS ${LIBSSH2_LDFLAGS})
-	#SET(LIBGIT2_PC_LIBS "${LIBGIT2_PC_LIBS} ${LIBSSH2_LDFLAGS}")
+	SET(LIBGIT2_PC_REQUIRES "${LIBGIT2_PC_REQUIRES} libssh2")
```

Comment:
I found the commit explaining why we do this: 790cabf. If we put them in requires, `pkg-config` will put *their* private libs in the linker args as well, which we don't want. We want to give tell whoever is embedding libgit2 which options we would have used, were we to create the shared library.